src/util.js

```diff
@@ -0,0 +1,37 @@
+/**
+  Utility function that wraps a function and provides two tracking functions
+  to allow you to track whether a function has been called
+  Type: (fn) -> fn
+**/
+var table = {};
```

Comment:
Seeing as how this isn't huge and archaic (like graphics), could it be written in Typescript instead?

Reply:
done